packages/sdk-v2/src/sdk/claim/react.ts

```diff
@@ -15,11 +15,14 @@ import useRefreshOrNever from "../../hooks/useRefreshOrNever";
 import { useGetContract, useGetEnvChainId, useReadOnlySDK, useSDK } from "../base/react";
 import { Envs, SupportedChains } from "../constants";
 
-export const useFVLink = () => {
-  const { chainId } = useGetEnvChainId();
+export const useFVLink = (requiredChainId?: number, whitelistAtChain: boolean = false) => {
+  const { chainId } = useGetEnvChainId(requiredChainId);
```

Comment:
```suggestion
  const { chainId } = useGetEnvChainId(requiredChainId);
  const { whitelistAtChain } = useContext(Web3Context);
```

Reply:
@L03TJ3 whitelistAtChain is the feature flag, we will set/unset it at the app side
it's needed for the case on some env we don't have new contracts deployed, so we could turn it off and ignore chainId value passed, calling old method receiving just 2 args instead of 4 (with chainid)

short example about WDIM: https://github.com/GoodDollar/GoodServer/commit/2f3f7f9a958f4c1697cca3feff712cdbcf993625#diff-57139424517d0125b1d2bba966555078d6141611aeb8c24a70555865ae3cf8c9R371

does we're keeping some feature flags at Web3Context ? if yes - then ok, please proceed with changes required

Reply:
we dont need a feature flag
backend is handling the chainId